templates/typescript-testing/test/restate_test_environment.ts

```diff
@@ -6,6 +6,7 @@ import {
     TestContainers,
     Wait,
 } from "testcontainers";
+import { tableFromIPC } from "apache-arrow";
```

Comment:
Bringing in apache arrow here is such a PITA :( 

I think down the road we should have an option in our `/query` endpoint to simply return json, as I do in the CLI...

Reply:
yes, i honestly wonder if i should just do this now